packages/cloud_firestore/cloud_firestore/pubspec.yaml

```diff
@@ -16,10 +16,10 @@ environment:
 
 dependencies:
   cloud_firestore_platform_interface: ^5.10.0
```

Comment:
shouldn't this bump to latest `cloud_firestore_platform_interface` version? `5.10.2`